plugin/general/editor.go

```diff
@@ -73,6 +74,20 @@ func editComposeFile(ctx *context.Context, file string, executorId string, taskI
 	return file, ports, err
 }
 
+func getVersion(filesMap types.ServiceDetail, file string) string {
+	if currentVersion, ok := filesMap[file][types.VERSION].(string); ok {
```

Comment:
Return "2.1" when the version is not found can avoid extra indent. 

````
ver, ok := xxx
if !ok { 
    return 2.1 
}

// found, cast etc.
````

Also better make default values string/float constant and use them in error message as well